frappe/public/js/frappe/views/communication.js

```diff
@@ -671,17 +674,27 @@ frappe.views.CommunicationComposer = Class.extend({
 			content = `
 				<div><br></div>
 				${reply}
-				<div class="ql-collapse" data-collapse="true">
-					<span data-comment='original-reply'>Reply To</span>
-					<blockquote>
-					<p>${__("On {0}, {1} wrote:", [frappe.datetime.global_date_format(communication_date) , last_email.sender])}</p>
-					${last_email_content}
-					</blockquote>
-				</div>
+				${this.separator_element}
+				<p>${__("On {0}, {1} wrote:", [frappe.datetime.global_date_format(communication_date) , last_email.sender])}</p>
+				<blockquote>
+				${last_email_content}
+				</blockquote>
 			`;
 		} else {
 			content = "<div><br></div>" + reply;
 		}
 		fields.content.set_value(content);
 	}
 });
+
+frappe.html2text = function html2text(html) {
```

Comment:
move this to `utils.js`?

Reply:
moved inline. This is specialized for the timeline (as it preserves whitespace)